src/components/UserSelector.tsx

```diff
@@ -75,6 +66,7 @@ export const UserSelector: React.FC<Props> = ({
               href={`#user-${user.id}`}
               onClick={() => {
                 onChange(user);
+                setExpanded(false);
```

Comment:
Pls move the function inside `onClick` handler to a separate const and use as a callback here

Reply:
I fixed all the errors what you asked me before, only i don`t have problems with infinite loading loop, everything working correctly by my main link and locally. I asked to check another mentor to check it because i wasn`t sure about it. He told everythnig looking good to him. I will share a screen from my desk also
![3](https://github.com/mate-academy/react_redux-list-of-posts/assets/114413825/a90bdc36-411c-4262-870f-e85a17bdbe7e)